packaging/conf/configuration.yml

```diff
@@ -35,9 +35,6 @@
 #################################################################
 
 default:
-  rails_cache_store: <%= ENV.fetch('RAILS_CACHE_STORE') { :memcache }.to_sym %>
```

Comment:
There are some references to `RAILS_CACHE_STORE` in documentation (related to docker) and in some docker-related files. Not sure if it should be updated too or not.

Reply:
I've opened https://community.openproject.org/work_packages/45416 for the Dockerfile, as there are more envs judging by a quick look at the dockerfile.

Reply:
Fixed in https://github.com/opf/openproject/commit/b91ec14e854c9d6202701f5ad537ef162c961fcc.